.gitignore

```diff
@@ -9,3 +9,4 @@ venv/
 *.pyc
 app/db.sqlite3
 db.sqlite3
+.venv
```

Comment:
There is an extra space at the end of the `.venv` entry. It should be removed to ensure that the `.venv` directory is correctly ignored.